src/components/i18nStrings.vue

```diff
@@ -135,11 +136,26 @@ export default {
 					this.$store.commit('cleanStateFor', this.path)
 				}
 			},
-      		deep: true
+			deep: true,
+			
+			"$store.state.languages": function(languages) {
+				for (const lang in languages) {
+					if (languages[lang]) {
+						this.addPair(lang)
+					}
+				}
+			},
 		},
 	},
 	created() {
 		this.state = this.value
+		const languages = this.$store.state.languages
```

Comment:
What is the javascript type of this.$store.state.languages in this line?

Reply:
object